django/db/backends/postgresql/schema.py

```diff
@@ -19,7 +19,8 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     sql_delete_index = "DROP INDEX IF EXISTS %(name)s"
     sql_delete_index_concurrently = "DROP INDEX CONCURRENTLY IF EXISTS %(name)s"
 
-    sql_create_column_inline_fk = 'REFERENCES %(to_table)s(%(to_column)s)%(deferrable)s'
+    sql_create_column_inline_fk = 'CONSTRAINT %(name)s REFERENCES %(to_table)s(%(to_column)s)%(deferrable)s'
+    sql_column_inline_fk_immediate = 'SET CONSTRAINTS %(name)s IMMEDIATE'
```

Comment:
It has to be switched back to `DEFERRED` afterwards

```suggestion
    sql_column_inline_fk_immediate = 'SET CONSTRAINTS %(name)s IMMEDIATE;SET CONSTRAINTS %(name)s DEFERRED'
```

Reply:
It won't work because an update is between adding a constraint and an index. Also I don't think that it's required because `SET CONSTRAINTS` change constraints only for the current transaction. :thinking: